src/main/java/com/nnaltakyan/core/auth/domain/verification/events/EventPublisher.java

```diff
@@ -0,0 +1,17 @@
+package com.nnaltakyan.core.auth.domain.verification.events;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class EventPublisher {
+    private final ApplicationEventPublisher eventPublisher;
+
+    public void publishEvent(SendVerificationEmailEvent emailEvent){
+        log.info("Email verification event has been thrown.");
```

Comment:
please add email in the log

Reply:
@nareknaltakyan1 At the point of publishing the event I only have "userId" property in email event.

Reply:
ok, just log userID